src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java

```diff
@@ -752,28 +752,30 @@ public class CicCase {
     private String firstDueDate;
 
     private LocalDate findEarliestDate(List<ListValue<DateModel>> dueDateList, LocalDate compare) {
-        LocalDate earliestDate = LocalDate.now();
+
```

Comment:
Hi Elif, Was trying to fix code smells here, as it was breaking pipeline, while not changing functionality...

SonarLint doesn't like you returning compare, since that's a parameter you've passed in.
So can you keep the local variable and return that, instead?

Reply:
done